### PR TITLE
[PERFSCALE-3337] Jetlag CI

### PR DESCRIPTION
**Pull request by josecastillolema**

### CODE REVIEW

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-redhat-performance-jetlag-main-deploy-5nodes

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-redhat-performance-jetlag-main-deploy-5nodes

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-redhat-performance-jetlag-main-deploy-5nodes

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-redhat-performance-jetlag-main-deploy-5nodes

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-redhat-performance-jetlag-main-deploy-5nodes

**Comment by rsevilla87** [comment]
/pj-rehearse pull-ci-redhat-performance-jetlag-main-deploy-5nodes pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-4.17-latest-x86-deploy-6nodes

**Comment by rsevilla87** [comment]
/pj-rehearse pull-ci-redhat-performance-jetlag-main-deploy-5nodes pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-deploy-6nodes

**Comment by rsevilla87** [comment]
/pj-rehearse pull-ci-redhat-performance-jetlag-main-deploy-5nodes pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-deploy-6nodes

**Comment by rsevilla87** [comment]
/pj-rehearse pull-ci-redhat-performance-jetlag-main-deploy-5nodes pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-deploy-6nodes

**Comment by rsevilla87** [comment]
/pj-rehearse pull-ci-redhat-performance-jetlag-main-deploy-5nodes pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-deploy-6nodes

**Comment by rsevilla87** [comment]
/pj-rehearse pull-ci-redhat-performance-jetlag-main-deploy-5nodes pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-deploy-6nodes

**Comment by rsevilla87** [comment]
/pj-rehearse pull-ci-redhat-performance-jetlag-main-deploy-5nodes pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-deploy-6nodes

File: core-services/prow/02_config/medik8s/_prowconfig.yaml
Comment (psalajova) [inline core-services/prow/02_config/medik8s/_prowconfig.yaml:1]: The changes in this file are a duplicate of what's already there, could you remove them? Sorry for inconvenience, it's a known bug

Comment (rsevilla87) [reply core-services/prow/02_config/medik8s/_prowconfig.yaml:1]: fixed, thanks for taking a look

**Comment by rsevilla87** [comment]
/pj-rehearse pull-ci-redhat-performance-jetlag-main-deploy-sno

File: core-services/prow/02_config/medik8s/_prowconfig.yaml
Comment (psalajova) [reply core-services/prow/02_config/medik8s/_prowconfig.yaml:1]: np

File: core-services/prow/02_config/generate-boskos.py [hunk 1]
```diff
@@ -84,6 +84,9 @@
     'metal-perscale-cpt-quota-slice': {
```
Comment (jtaleric) [inline core-services/prow/02_config/generate-boskos.py:84]: perf*

Comment (josecastillolema) [reply core-services/prow/02_config/generate-boskos.py:84]: Its perscale, there is a typo unfortunately in ci-tools

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-redhat-performance-jetlag-main-deploy-sno

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-redhat-performance-jetlag-main-deploy-sno

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-redhat-performance-jetlag-main-deploy-sno

**Pull request merged**